docs/v2-release-notes.md

```diff
@@ -2,6 +2,16 @@
 
 This page summarizes changes for each major update in Timeplus Enterprise v2 and Timeplus Proton, including new features and important bug fixes.
 
+## Jul 22, 2024
+
+### Timeplus Proton v1.5.14
```

Comment:
missed Timeplus Proton v1.5.13
* fix: properly clear the format schema cache when the format schema is dropped
* fix: a null pointer access before type cast